homeassistant/components/motion_blinds/cover.py

```diff
@@ -174,7 +174,7 @@ class MotionBaseDevice(MotionCoordinatorEntity, CoverEntity):
 
     _restore_tilt = False
 
-    def __init__(self, coordinator, blind, device_class):
```

Comment:
What about `is_closed()`and `current_cover_position()`, do they need reversing as well? If not then at least the docstring still seems wrong.